mds/serializers.py

```diff
@@ -230,12 +242,64 @@ class AreaSerializer(BaseModelSerializer):
         required=False, help_text="Unique Area identifier (UUID)"
     )
     label = serializers.CharField(help_text="Name of the Area")
-    creation_date = serializers.DateTimeField(help_text="Area creation date")
+    creation_date = serializers.DateTimeField(
+        required=False, help_text="Area creation date"
+    )
     deletion_date = serializers.DateTimeField(
-        required=False, help_text="Area deletion date"
+        required=False, allow_null=True, help_text="Area deletion date"
+    )
+    properties = serializers.JSONField(
+        required=False, help_text="Properties of the Area"
     )
     polygons = PolygonSerializer(required=False, many=True)
 
+    def create(self, validated_data):
+        data = validated_data
+        polygons = data.pop("polygons", None)
+        instance = models.Area(**data)
+        instance.save()
+        if polygons is not None and len(polygons) > 0:
+            for polygon in [dict(p) for p in polygons]:
+                polygon_instance = models.Polygon.objects.filter(
+                    id=polygon["id"]
+                ).first()
+                instance.polygons.add(polygon_instance)
+            instance.save()
+        return instance
+
+    def update(self, instance, validated_data):
```

Comment:
It could be interesting to keep the history of an area (e.g. to price something that occured before the upate).